src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -541,6 +541,19 @@
 	 * @return The emoji with the given name (or null if one was not found).
 	 */
 	IEmoji getEmojiByName(String name);
+	
+	/**
+ 	 * This adds a custom emoji to the guild.
```

Comment:
The documentation, word wise, should be consistent with something similar to createChannel. For instance, the description should be Creates a new emoji., the return The new emoji., etc.